getting-started/multiband-analysis.rst

```diff
@@ -145,10 +146,23 @@ Then make an index array from the combination of ``detect_isPatchInner`` and ``d
 
    inInnerRegions = refTable['detect_isPatchInner'] & refTable['detect_isTractInner']
 
+Rejecting sky objects
+--------------------------
+
+Some sources in the catalog may be "sky" objects, if ``detectCoaddSources.py`` is configured to add them.  These objects do not correspond to a detection, but are
```

Comment:
Suggested edit:

```
``detectCoaddSources.py`` is configured, by default, to add "sky" objects to the catalog.
These "sky" objects do not correspond to detections but are used for characterizing the image's noise properties.

The ``merge_peak_sky`` flag identifies these "sky" objects:
```

(please verify that adding sky sources is the default configuration)

And again, ensure that you write one sentence per line, not hard-wrapping the text.